### PR TITLE
Remove combot dependency

### PR DESCRIPTION
**Pull request by selBaez**
Merging contributions across branches

### CODE REVIEW

**Comment by piekvossen** [comment]
I need to focus on teaching now. Can someone else test the  request?

**Comment by tae898** [comment]
At the directory `examples`, when I run `python jsonld.py`, it throws an error 
```
  File "jsonld.py", line 19, in <module>
    jsonld_context = brain.get_jsonld_context(save_path=jsonld_filepath)
  File "/home/tk/repos/cltl-knowledgerepresentation/src/cltl/brain/basic_brain.py", line 192, in get_jsonld_context
    with open(save_path, 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/tk/repos/cltl-knowledgerepresentation/cltl/brain/ontologies/integration_context.jsonld'
```
The other python scripts run okay, but time to time it gives me a warning `Failed to query Wikidata`.

File: tests/carl.py [hunk 1]
```diff
@@ -58,7 +58,9 @@
 if __name__ == "__main__":
 
     # Create brain connection
-    log_path = pathlib.Path.cwd().parent / 'src' / 'cltl' / 'brain' / 'logs'
+   # Piek: parent is the folder above the project, had to take it out
+   # log_path = pathlib.Path.cwd().parent / 'src' / 'cltl' / 'brain' / 'logs'
+    log_path = pathlib.Path.cwd() / 'src' / 'cltl' / 'brain' / 'logs'
```
Comment (numblr) [inline tests/carl.py]: Can't we just use a temp folder for this?

File: src/cltl/combot/infra/di_container.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+from threading import Lock
+from time import sleep
+
+_MAX_WAIT = 1000
+
+
+class DIContainer(object):
```
Comment (numblr) [inline src/cltl/combot/infra/di_container.py]: If you are OK with that I can take a look and remove the usage of this entirely (outside of this PR..).

File: tests/carl.py (hunk 1 above)
Comment (numblr) [reply tests/carl.py]: OK, I made it a command line parameter and use a temporary directory by default (which is cleaned up after running the script).

Comment (tae898) [reply tests/carl.py]: Some warning/error messages while running  `for example in *.py; do python $example; done`

```
Failed to query Wikidata
```

```
Traceback (most recent call last):
  File "jsonld.py", line 19, in <module>
    jsonld_context = brain.get_jsonld_context(save_path=jsonld_filepath)
  File "/home/tk/repos/cltl-knowledgerepresentation/src/cltl/brain/basic_brain.py", line 192, in get_jsonld_context
    with open(save_path, 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/tk/repos/cltl-knowledgerepresentation/cltl/brain/ontologies/integration_context.jsonld'
```

Comment (tae898) [reply tests/carl.py]: haha sorry I forgot to do `git pull`. The `jsonld.py` error doesn't happen anymore.

**Review by tae898 — APPROVED** [review]

**Review by numblr — APPROVED** [review]

**Pull request merged**